packages/validator/src/validators/anyValidator.ts

```diff
@@ -0,0 +1,88 @@
+import * as utils from '../utils/utils';
+import { processSpecs } from '../processor';
+import { Log, Roots } from '../types';
+import * as logger from '../utils/logger';
+
+/**
+ * Checks if at least one param exists in provided specification conforming the validator specification structure
+ * @param specs - specification that is being validated
+ * @param template - must be on the same level as specs
+ * @param paramPath - string of parameters separated by ".", representing path to current specs location
+ * @param nonRedundantParams - object containing all required and optional parameters that are being used
+ * @param roots - roots of specs and nonRedundantParams
+ * @param templatePath - path to current validator template file
+ * @returns empty array if at least one child of specs satisfies template, otherwise returns array with error message
+ */
+export function validateAny(
+  specs: any,
+  template: any,
+  paramPath: string[],
+  nonRedundantParams: any,
+  roots: Roots,
+  templatePath: string
+): Log[] {
+  if (!specs || typeof specs !== 'object') {
+    return [logger.error(`Required conditions not met in ${paramPath.join('.')}`)];
```

Comment:
You have a function for this message in `messages.ts`

Reply:
Fixed in 0529c86051a72a666d1be1f8107b3a3fb80d3cff